assets/directory_asset_bundle.h

```diff
@@ -14,17 +14,21 @@ namespace flutter {
 
 class DirectoryAssetBundle : public AssetResolver {
  public:
-  explicit DirectoryAssetBundle(fml::UniqueFD descriptor);
+  DirectoryAssetBundle(fml::UniqueFD descriptor, bool should_preserve);
```

Comment:
rename the param here to match the impl/ivar name.

Reply:
Fixed